rdkit/Chem/mol2.py

```diff
@@ -0,0 +1,156 @@
+from rdkit.Chem.rdPartialCharges import ComputeGasteigerCharges
+
+def _get_positions(mol,confId=-1):
+    conf = mol.GetConformer(confId)
+    return [conf.GetAtomPosition(i) for i in range(conf.GetNumAtoms())]
+
+
+def MolToMol2File(mol, filename, confId=-1):
+    """Writes a Mol2 file for a molecule
+      ARGUMENTS:
+
+        - mol: the molecule
+        - filename: the file to write to
+        - confId: (optional) selects which conformation to output (-1 = default)
+                  if set to None will return all conformers
+
+      RETURNS:
+
+        None
+    """      
+    block = MolToMol2Block(mol, confId)
+    open(filename, "w").writelines(block)
+    
+def MolToMol2Block(mol, confId=-1):
+    """Returns a Mol2 string block for a molecule
+      ARGUMENTS:
+
+        - mol: the molecule
+        - confId: (optional) selects which conformation to output (-1 = default)
+                  if set to None will return all conformers
+
+      RETURNS:
+
+        a string
+    """  
+
+    #
+    # References 
+    # - Format specs http://www.tripos.com/data/support/mol2.pdf
+    #
+    
+    confIds = (confId,)
+    
+    if confId == None:
+        confIds = Chem.Mol.GetNumConformers()
+    
+    blocks = []
+    
+    # compute charges
+    ComputeGasteigerCharges(mol)
+    
+    for confId in confIds:
+        
+        molecule = """@<TRIPOS>MOLECULE
+{}
+{} {} 0 0 0
+SMALL
+GASTEIGER\n\n""".format(mol.GetProp("_Name") if mol.HasProp("_Name") else "UNK", mol.GetNumAtoms(), mol.GetNumBonds())
+
+        # FIXME "USER_CHARGES" could become 'Gasteiger charges'
+        # FIXME "SMALL" means small molecule but could become "PROTEIN"
+        
+        pos = _get_positions(mol, confId)
+        atom_lines = ["{:>4} {:>4} {:>13.4f} {:>9.4f} {:>9.4f} {:<5} {} {} {:>7.4f}".format(a.GetIdx()+1, 
+                                                                                                         a.GetSymbol(), 
+                                                                                                         float(pos[a.GetIdx()][0]), float(pos[a.GetIdx()][1]), float(pos[a.GetIdx()][2]), 
+                                                                                                         _sybyl_atom_type(a), 
+                                                                                                         1, "UNL", 
+                                                                                                         float(a.GetProp('_GasteigerCharge').replace(',','.')) if a.HasProp('_GasteigerCharge') else 0.0) for a in mol.GetAtoms()]
+        atom_lines = ["@<TRIPOS>ATOM"] +  atom_lines + ["\n"]
+        atom_lines = "\n".join(atom_lines)
+            
+        bond_lines = [ "{:>5} {:>5} {:>5} {:>2}".format(bid+1, b.GetBeginAtomIdx()+1, b.GetEndAtomIdx()+1,  "ar" if int(b.GetBondType()) > 2 else "am" if _amide_bond(b) else str(int(b.GetBondType())) ) for bid, (b) in enumerate(mol.GetBonds()) ]
```

Comment:
It looks like this writes triple bonds into the mol2 block as aromatic bonds. I don't think that's correct.